lms/envs/devstack_docker.py

```diff
@@ -30,8 +30,10 @@
     'ENABLE_COURSEWARE_SEARCH': False,
     'ENABLE_COURSE_DISCOVERY': False,
     'ENABLE_DASHBOARD_SEARCH': False,
-    'SHOW_LANGUAGE_SELECTOR': True,
```

Comment:
Yes, the idea is that this setting should be deprecated in favor of the new LANGUAGE_SELECTOR setting. Thinking about it, I can probably just remove it from the other config files as well, so long as the code continues to honor the original setting if it is set on an older installation.